app/helpers/authentication_helper.rb

```diff
@@ -24,4 +24,12 @@ def recaptcha_configured_and_enabled?
       SiteConfig.recaptcha_site_key.present? &&
       SiteConfig.require_captcha_for_email_password_registration
   end
+
+  def forem_creator_flow_enabled?
+    Flipper.enabled?(:creator_onboarding) && waiting_on_first_user?
+  end
+
+  def waiting_on_first_user?
+    SiteConfig.waiting_on_first_user
```

Comment:
The config gem automatically generates predicate methods for boolean attributes, so `    SiteConfig.waiting_on_first_user?` works too. Maybe we don't need this wrapper method?